src/dashboard/Data/Browser/AttachRowsDialog.react.js

```diff
@@ -0,0 +1,95 @@
+import * as Filters from 'lib/Filters';
+import Modal from 'components/Modal/Modal.react';
+import React from 'react';
+import Field from 'components/Field/Field.react';
+import Label from 'components/Label/Label.react';
+import TextInput from 'components/TextInput/TextInput.react';
+import { List, Map } from 'immutable';
```

Comment:
This component doesn't need immutable structures. Please remove this before squashing.